### PR TITLE
Use ChartStore cache data for GetStats API. The time series data is r…

### PR DESCRIPTION
**Pull request by shifucun**
…anked by import name and measurement method

### CODE REVIEW

File: store/populations.go [hunk 1]
```diff
@@ -28,11 +28,26 @@ import (
...
+	mmethod string
+}
+
+var statsRanking = map[rankKey]int{
+	rankKey{"CensusPEP", "CensusPEPSurvey"}:                   0,
+	rankKey{"CensusACS5YearSurvey", "CensusACS5yrSurvey"}:     1,
+	rankKey{"EurostatData", "EurostatRegionalPopulationData"}: 2,
+	rankKey{"WorldDevelopmentIndicators", ""}:                 3,
+	rankKey{"BLS_LAUS", "BLSSeasonallyUnadjusted"}:            0,
+	rankKey{"EurostatData", ""}:                               1,
```
Comment (pradh) [inline store/populations.go:46]: Add some comments on what the ranking means?

Also, add inline comments to demarcate
  // Population
  // Unemployment Rate
?

File: store/populations.go [hunk 2]
```diff
@@ -493,38 +508,32 @@ func getObsSeries(
...
+		result := &pb.ObsTimeSeries{}
+		result.Unit = x.ObsTimeSeries.GetUnit()
+		result.PlaceName = x.ObsTimeSeries.GetPlaceName()
+		result.IsDcAggregate = x.ObsTimeSeries.GetIsDcAggregate()
+		bestScore := lowestRank
+		for _, series := range x.ObsTimeSeries.SourceSeries {
+			key := rankKey{series.GetImportName(), series.GetMeasurementMethod()}
+			score, ok := statsRanking[key]
+			if !ok {
+				score = lowestRank
```
Comment (pradh) [inline store/populations.go:524]: nit: If not found, should we consider it having the highestRank, in that it will always be shown :)

Comment (pradh) [reply store/populations.go:524]: Hmm, never mind. This is the correct thing to do when you have new sources for the same statvar.

File: store/populations.go (hunk 1 above)
Comment (shifucun) [reply store/populations.go]: Done

**Pull request merged**